### ISSUE TITLE
Remove expected time to win from the GUI

### ISSUE THREAD

**Issue by shaneo257** [Chia-Network/chia-blockchain#1410]
Hi Chia Team,

Can we please remove the Expected Time to Win, as its causing much angst with the people on keybase, as they're expecting to get a block every x hours/days it says, when we know its a lottery and its a rough guesstimate

**Comment by mariano54** [comment]
It's pretty useful for people to get a sense of how long it could take. Otherwise people with 10 plots will be complaining constantly. Maybe we need to improve the messaging and say that there is a 50% chance that you will win a block by this time.. or something like that

**Comment by shaneo257** [comment]
I agree, but I'm sure you can see on keybase many people are complaining
Maybe even change it to Estimated time to win a block, instead of expected, because you know, millennials

**Comment by mattselkey** [comment]
not a millennial here ;)
But for what it's worth, changing this in the gui makes a lot of sense.

having days which are clearly in no way indicative of the real time to win is going to give a poor user experience. i.e user waits 20 days, total network space increase, days to win increases, repeat.

I'd suggest changing it to "chances of winning" and make it probability based i.e 1 in a 500, 1 in a million, etc, also set a disclaimer that it will fluctuate and until pools are around people with less than 20TB of plots don't have a snowballs chance in hell 😂

**Comment by fiveangle** [comment]
I'd remind them that "expected" != "guarenteed" and point them [here ](https://arxiv.org/pdf/1404.0834.pdf) so hey can feverishly review the contents and hopefully they will dizzy and take a nap before considering returning to the keybase chat 😝
